.github/workflows/offline-link-check.yml

```diff
@@ -10,9 +10,10 @@ jobs:
   link_check:
     runs-on: ubuntu-latest
     steps:
-      uses: actions/checkout@v5
-      uses: lycheeverse/lychee-action@v2
-        # Only check local files, no network requests
-        args: --offline --no-progress 'src/content/docs/**/*.md'
-        # Fail job when lychee returns a non-zero exit code
-        fail: true
+      - uses: actions/checkout@v5
+      - uses: lycheeverse/lychee-action@v2
+        with:
+          # Only check local files, no network requests
+          args: --config './lychee.toml' --no-progress --offline --root-dir "$GITHUB_WORKSPACE/public/" 'src/content/docs/**/*.md'
```

Comment:
@JimMadge i think you are fixing something that's been bugging us at pyOpenSci for a while. Every time we add a new page to our site, lychee breaks. So this forces lychee to check links "internally" vs online . is that correct?

Thank you for this!!